src/serverlessFunctions/parseFromTemplate.js

```diff
@@ -23,8 +23,32 @@ const buildFnPathData = (path) => {
   };
 };
 
+const buildLayersConfig = (basePath, templatePath, template, prelimLayers) => {
+  const config = [];
+
+  if (prelimLayers && prelimLayers.length > 0) {
+    prelimLayers.forEach((layer) => {
+      if (!('Ref' in layer)) throw new Error('Only referenced layers are supported.');
+
+      const layerName = layer.Ref;
+      const cftLayer = template.Resources[layerName];
+      const layerContentUri = cftLayer.Properties.ContentUri;
```

Comment:
According to the [docs](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-resource-layerversion.html#sam-layerversion-contenturi), `ContentUri` might be an object, which will cause the following line to break as we try to merge strings with an object.

Should we apply some validation?

Reply:
Also a very good catch. I will only only this layer to be processed if it's ContentUri is a relative path (i.e. string)